lib/debug/websocket.dart

```diff
@@ -0,0 +1,56 @@
+import 'dart:async';
+import 'dart:js';
+
+import 'package:atom/node/node.dart';
+
+class WebSocket {
```

Comment:
At some point in the future, should we move this class into the `atom.dart` plugin?